tools/docker/env/Dockerfile

```diff
@@ -35,6 +34,18 @@ RUN mkdir -p /syzkaller/gopath/src/github.com/google/syzkaller && \
 	mkdir -p /syzkaller/.cache && \
 	chmod -R 0777 /syzkaller
 
+# The default clang-11 is too old, install the latest one.
+RUN apt-get install -y -q gnupg software-properties-common apt-transport-https
```

Comment:
We need to remove this from big-env b/c it's derived from env.

Somebody proposed to merge all of env/big-env/syzbot b/c this separation does not seem to be any useful. Just mentioning, don't do it in this change.